firmware/arduino/chessboard/chessboard.ino

```diff
@@ -38,7 +38,7 @@ int yMotor[5] = {Y_MOTOR_STEP, Y_MOTOR_DIR, Y_MOTOR_MS1, Y_MOTOR_MS2, Y_AXIS_END
 // ================================
 
 // Distance definitions
-#define MILLIMETERS_PER_SQUARE 63
+#define MILLIMETERS_PER_SQUARE 32
```

Comment:
So, this is definitely the first step in the process of implementing this. But ask yourself, what's the effect of doing this with the current code? The answer is that is just reduces the size of each space. However, in this new scheme, one whole piece space will be two unit spaces wide and tall. So, with that in mind, how can we change the code to make one row or col equal two unit spaces?